assets/js/base/components/tabs/index.js

```diff
@@ -21,6 +21,7 @@ const Tabs = ( {
 	instanceId,
 	ariaLabel = __( 'Tabbed Content', 'woo-gutenberg-products-block' ),
 	children,
+	id,
```

Comment:
Should we generate a default fallback for ID like on `selectedId`.

Reply:
This is mostly just an optional id for the entire tabs container. There's an issue (#1474) for reviewing/improving/replacing this tabs component I added, when we get to that at some point, we can use the opportunity to review whether this id is even needed (or improve).